daphne/src/testing.rs

```diff
@@ -95,6 +109,11 @@ impl MockAggregator {
         // Just use the first key in the hash map.
         self.tasks.keys().next().as_ref().unwrap()
     }
+
+    #[cfg(test)]
+    pub fn get_report_store(&self) -> &ReportStore {
+        &self.report_store
```

Comment:
you could avoid having to add this method by making `report_store` itself `pub(crate)`:

```rust

pub(crate) struct MockAggregator {
    tasks: HashMap<Id, DapTaskConfig>,
    hpke_receiver_config_list: Vec<HpkeReceiverConfig>,
    pub(crate) report_store: ReportStore,
    helper_state_store: HelperStateStore,
}
```

Reply:
Thank you, I totally missed this.